app/renderer/components/styles/theme.js

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ /**
+  * Includes color options for theming
+  * This should be used as a boilerplate for all
+  * future theming, including darkUI.
+  * Note: If an element is not color-related, it should go into global.js
+  */
+  module.exports.theme = {
+    tab: {
+      // mimics chrome hover effect
+      transition: `
+        background-color 200ms cubic-bezier(0.26, 0.63, 0.39, 0.65),
+        color 200ms linear
+      `,
+      background: '#ddd',
+      borderColor: '#bbb',
+      color: '#000',
+      hover: {
```

Comment:
would you mind adding lines for readability?

Reply:
Confirmed this was addressed on https://github.com/brave/browser-laptop/blob/83864a8769ee10cb2d2d864b19289ba5e229bd34/app/renderer/components/styles/theme.js